### ISSUE TITLE
Hide decorative user SVGs in the share modal

### ISSUE THREAD

**Issue by Ovgodd** [suitenumerique/docs#2316]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**

In the share modal, the `<svg>` elements used to illustrate each document user are currently exposed to assistive technologies.

These SVGs are decorative. They do not provide additional useful information, because the user information is already available through the visible text.

As a result, screen reader users may hear unnecessary decorative content while navigating the modal.

**Describe the solution you'd like**

Mark these decorative SVGs as hidden from assistive technologies.

Example:

```html
<svg aria-hidden="true">
  ...
</svg>
```

The user name and related information should remain available through text content.

**Describe alternatives you've considered**

**Discovery, Documentation, Adoption, Migration Strategy**

This should be checked in the share modal, especially in the list of users who have access to the document.

**Do you want to work on it through a Pull Request?**

Yes.

**Comment by Ovgodd** [comment]
Closing this ticket for the other one : [2332](https://github.com/suitenumerique/docs/issues/2332)
